src/book/book.service.ts

```diff
@@ -25,6 +25,85 @@ export class BookService {
     }
     return book
   }
+
+  async getAllBooks(filters: any, page: number = 1, limit: number = 20) {
```

Comment:
[REQ] Please add a type for the filters, something like:
```.ts
type Filters = {
  title?: string
  authorId?: string
  authorName?: string
  isbn?: string
  category?: string
  minPrice?: string
  maxPrice?: string
}
```
```suggestion
  async getAllBooks(filters: Filters, page: number = 1, limit: number = 20) {
```